components/LoadingScreen/index.jsx

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import Skeleton, { SkeletonTheme } from 'react-loading-skeleton';
+import 'react-loading-skeleton/dist/skeleton.css';
+
+export const BASE_COLOR = '#313236';
+export const HIGHLIGHT_COLOR = '#49494D';
+export const BORDER_RADIUS = 12;
+
+export const loadingScreenTestIds = {
+  base: 'loading-screen',
+  skeleton: 'loading-screen__skeleton',
+  skeletonTheme: 'loading-screen__skeleton-theme',
+};
+
+const LoadingScreen = ({ width, height, circle = false, className }) => {
```

Comment:
How about renaming this component to `Loading` or `Loader` to avoid confusion as it is not a screen? 😅

Reply:
sure, what about renaming it to `LoadingSkeleton` then, to keep up with the library used(`react-skeleton`)?

Reply:
That's sounds fine too 👍🏼